src/app/objects/resume.ts

```diff
@@ -0,0 +1,57 @@
+import { template } from "./template";
+
+export interface resume {
+  template: template,
+  header: header,
+  workExps: workExp[],
+  educations: education[],
+  certificates?: string[],
+  skills: string[],
+  summary: string
+}
+
+export interface header {
+  id: number;
+  firstName: string;
+  lastName: string,
+  profession: string,
+  email: string,
+  phone: string,
+  address: {
```

Comment:
For address, I feel it works better as a string. More free-flowing and less limiting for different cultures/countries

Reply:
However we require users to fill multiple inputs for address information, so we can achieve country/province/city selection in the future instead of text input for better experience. Do we want to use a single text input for whole address & postal code?
e.g resumegenius:
![1714011643458](https://github.com/P311/zero-one/assets/33985370/1b406875-8dae-4381-a81d-a72452c30cdf)
![image](https://github.com/P311/zero-one/assets/33985370/b7bab03a-d633-4dbc-8a10-69ca23830cca)